### PR TITLE
Some janitoring

### PR DESCRIPTION
**Pull request by akva2**
Quell a clang warning, add a missing include, fix namespace usage.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/material/fluidsystems/ThreeComponentFluidSystem.hh [hunk 1]
```diff
@@ -64,14 +65,14 @@ namespace Opm {
...
+        using Comp1 = C1<Scalar>;
+        using Comp2 = C10<Scalar>;
 
         template <class ValueType>
-        using ParameterCache = Opm::PTFlashParameterCache<ValueType, ThreeComponentFluidSystem<Scalar>>;
-        using ViscosityModel = typename Opm::ViscosityModels<Scalar, ThreeComponentFluidSystem<Scalar>>;
-        using PengRobinsonMixture = typename Opm::PengRobinsonMixture<Scalar, ThreeComponentFluidSystem<Scalar>>;
+        using ParameterCache = PTFlashParameterCache<ValueType, ThreeComponentFluidSystem<Scalar>>;
+        using ViscosityModel = typename ::Opm::ViscosityModels<Scalar, ThreeComponentFluidSystem<Scalar>>;
+        using PengRobinsonMixture = typename ::Opm::PengRobinsonMixture<Scalar, ThreeComponentFluidSystem<Scalar>>;
```
[the remark this answers is not in the document]
Comment (akva2) [reply opm/material/fluidsystems/ThreeComponentFluidSystem.hh]: absolutely.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me and I'll merge into master.

**Pull request merged**